product-service/helpers/validateProductData.ts

```diff
@@ -0,0 +1,24 @@
+export const validateProductData = (data) => {
+  let status = true;
+  let message = [];
+  if (!data.title) {
+    status = false;
+    message.push('title field is required');
+  }
+  if (!data.description) {
```

Comment:
I suppose that you can make this field optional

Reply:
Fixed.